charms/kserve-controller/tests/integration/test_charm.py

```diff
@@ -16,6 +16,9 @@
 
 logger = logging.getLogger(__name__)
 
+ISTIO_VERSION = "1.16/stable"
```

Comment:
@DnPlas Do we want to use `latest/edge` here?
There were updates to Istio that might solve integration test problems.

Reply:
I tried latest/edge first, then changed it to 1.16/stable to see if that would fix the errors, but it did not.